lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -84,6 +89,8 @@ def purchase(money, payment, options = {})
           if payment.is_a?(ApplePayPaymentToken)
             r.process { tokenize_apple_pay_token(payment) }
             payment = StripePaymentToken.new(r.params["token"]) if r.success?
+          elsif ach?(payment)
```

Comment:
Hm, agreed! Keeping that inside the `MultiResponse` is a holdover from when I was trying to handle bank accounts directly in the `purchase` method.

Returning a new Response is something we do elsewhere in the Stripe gateway, e.g. https://github.com/sdball/active_merchant/blob/add-ach-support-to-stripe/lib/active_merchant/billing/gateways/stripe.rb#L154